packages/Syntax/src/Syntax.tsx

```diff
@@ -3,21 +3,29 @@ import PropTypes from 'prop-types';
 import { cx, css } from '@leafygreen-ui/emotion';
 import hljs from 'highlight.js/lib/highlight';
 import { Variant, SupportedLanguages, Lang } from './types';
-import './globalStyles';
+
 const SupportedLanguagesList = Object.values(SupportedLanguages);
 
-SupportedLanguagesList.forEach(language => {
-  hljs.registerLanguage(
-    language,
-    require(`highlight.js/lib/languages/${language}`),
-  );
-});
+let syntaxHighlightingInitialized = false;
+
+function initializeSyntaxHighlighting() {
+  syntaxHighlightingInitialized = true;
 
-hljs.configure({
-  languages: SupportedLanguagesList,
-  classPrefix: 'lg-highlight-',
-  tabReplace: '  ',
-});
+  require('./globalStyles');
```

Comment:
Importing a module shouldn't have a side effect. Let's export a function that must be invoked to take on the desired action.